torch/csrc/jit/codegen/cuda/partition.cpp

```diff
@@ -74,8 +106,8 @@ inline bool isFusibleNode(const Node* node) {
   // Check we have a parsing rule
   bool isFusible = isNodeParsible(node);
   // Check if we have a tensor type it's one we support
-  isFusible = isFusible && allCompatableTensorTypes(node->inputs());
-  isFusible = isFusible && allCompatableTensorTypes(node->outputs());
+  isFusible = isFusible && checkInputTensorTypes(node);
```

Comment:
Have to make these changes so we won't exclude BN for its input/output tensor scalar types.